source/reference/methods.txt

```diff
@@ -373,6 +373,62 @@ Connection Methods
 
      - Description
 
+   * - :method:`Mongo()`
+
+     - JavaScript constructor to instantiate a database connection from
+       the ``mongo`` shell or from a JavaScript file.
+
+       The :method:`Mongo()` method takes a parameter called
+       ``ClientSideFieldLevelEncryptionOptions``, with the following options:
+
+       .. list-table::
+          :widths: 15 20 40
+          :header-rows: 1
+
+          * - Option
+
+            - Type
+
+            - Description
+
+          * - ``keyVaultClient``
+
+            - :method:`Mongo()` connection object
+
+            - *(Optional)* The MongoDB cluster hosting the key vault
+              collection. Omit to use the current database connection
+              as the key vault host.
+
+          * - ``keyVaultNamespace``
+
+            - String
+
+            - *(Required)* The full namespace of the key vault collection.
+
+          * - ``kmsProvider``
+
+            - Document
+
+            - *(Required)* The Key Management Service (KMS) used by
+              client-side field level encryption for managing a Customer
+              Master Key (CMK).
+
+          * - ``schemaMap``
+
+            - Document
+
+            - *(Optional)* The automatic client-side field level encryption
+              rules specified using the JSON schema Draft 4 standard syntax
+              and encryption-specific keywords.
+
+          * - ``bypassAutoEncryption``
+
+            - Boolean
+
+            - *(Optional)* Specify ``true`` to bypass automatic client-side
```

Comment:
How about: The flag to bypass automatic client-side ...

Reply:
Unable to comment below, but was wondering if you can fix the link tor Change Streams below. It should be :manual:`change stream cursor </changeStreams>` instead of :manual:`change stream cursor </change-streams>`?